swarm/pss/pss.go

```diff
@@ -1034,3 +1047,10 @@ func (p *Pss) digestBytes(msg []byte) pssDigest {
 	copy(digest[:], key[:digestLength])
 	return digest
 }
+
+func checkAddress(addr *PssAddress) bool {
+	if len(*addr) > addressLength {
```

Comment:
Signature of caller is:
```
func (p *Pss) SetPeerPublicKey(pubkey *ecdsa.PublicKey, topic Topic, address *PssAddress) error {
```

Actually `PssAddress` is `[]byte`, so signature should be `PssAddress`, and not `*PssAddress`. Currently it is possible to pass `nil` and panic here with nil deference.